.devcontainer/scripts/load-aliases.sh

```
@@ -1,121 +1,233 @@
+    else
+      unset "$_ca_var"
+    fi
+  fi
+done
+unset _ca_var _val
 
-netbox-run-bg() {
-  "$PLUGIN_DIR/.devcontainer/scripts/start-netbox.sh" --background
-}
+# Load shared process management helpers
+if ! source "$PLUGIN_DIR/.devcontainer/scripts/process-helpers.sh"; then
```

Shellcheck (0.11.0)

[info] Not following: ./.devcontainer/scripts/process-helpers.sh was not specified as input (see shellcheck -x).

(SC1091)